pyproject.toml

```diff
@@ -14,16 +14,17 @@ documentation = "https://slowapi.readthedocs.io/en/latest/"
 include = ["slowapi/py.typed"]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
 limits = "^1.5"
 redis = "^3.4.1"
+anyio = "3.0.0"
```

Comment:
Also question, perhaps about python development in general, but what's the reason for not including `starlette` as a py dependency? It is definitely required, after all.

Reply:
It's a very good question, and I don't think I have a good answer to it :)
I can't see moving `starlette` up here being a problem in any way.